### PR TITLE
Adding a new integration : virtual_thermostat

### PR DESCRIPTION
**Pull request by DB-CL**
**Description:**

Add the documentation for a new integration : `virtual_thermostat` . Please feel free to correct my english, or to give me some advice to be more respectful of the standards.

**Pull request in home-assistant (if applicable):** https://github.com/home-assistant/home-assistant/pull/27833

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by frenck** [comment]
@DB-CL Could you point me to the parent PR in our code base?

File: source/_integrations/virtual_thermostat.markdown [hunk 1]
```diff
@@ -0,0 +1,183 @@
+---
+title: "Virtual Thermostat"
+description: "A virtual thermostat implementation"
+logo: home-assistant.png
+ha_category:
+  - Climate
+ha_release: 0.1XX
```
Comment (klaasnicolaas) [inline source/_integrations/virtual_thermostat.markdown]:
```suggestion
ha_release: 0.101
```

File: source/_integrations/virtual_thermostat.markdown [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+ha_iot_class: Local Polling
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_integrations/virtual_thermostat.markdown]: Please remove the `redirect_from`

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/virtual_thermostat.markdown (hunk 2 above)
Comment (DB-CL) [reply source/_integrations/virtual_thermostat.markdown]: Done

**Comment by DB-CL** [comment]
I did the suggested changes and I updated the initial post with my PR

Let me know if you need something else.

**Pull request closed without merging**